src/saving/SaveInformation.cs

```diff
@@ -50,6 +55,7 @@ public enum SaveType
     public Guid ID { get; set; } = Guid.NewGuid();
 
     public SaveType Type { get; set; } = SaveType.Manual;
+    public string TypeString => GetDescriptionForSaveType(Type);
```

Comment:
Needs to be marked non json property.

Reply:
I think readonly props are by default ignored by NewtonsoftJson, but I'm not 100% sure. I'll add it for clarity sake

Reply:
I *think* I've seen just get properties get saved in json, because they are readable, on load they are probably automatically ignored.